packages/wdio-sync/src/utils.ts

```diff
@@ -18,25 +18,31 @@ export function sanitizeErrorMessage (commandError, savedError) {
     const err = new Error(message)
     err.name = name
 
-    let stackArr = savedError.stack.split('\n')
+    let stackArr: string | string[] | undefined = savedError.stack
+
+    if (stackArr) {
+        stackArr = stackArr.split('\n')
+    }
 
     /**
      * merge stack traces if `commandError` has stack trace
      */
     if (stack) {
         // remove duplicated error name from stack trace
         stack = stack.replace(`${err.name}: ${err.name}`, err.name)
-        // remove first stack trace line from second stack trace
-        stackArr[0] = '\n'
-        // merge
-        stackArr = [...stack.split('\n'), ...stackArr]
+        if (stackArr && stackArr.length) {
```

Comment:
```suggestion
        if (Array.isArray(stackArr)) {
```

Reply:
I believe you can remove `as` in line 35 `(stackArr as string[])` after the change

Reply:
well spotted, thanks!